lib/global.gi

```diff
@@ -199,6 +217,27 @@ InstallGlobalFunction( MakeReadWriteGlobal,
     MAKE_READ_WRITE_GLOBAL( name );
 end);
 
+#############################################################################
+##
+#F  MakeConstantGlobal ( <name> )  . . . . make a global variable constant
+##
+##  MakeConstantGlobal ( <name> ) marks the global variable named
```

Comment:
Why the weird wrapping here?

Reply:
once again, same as the wrapping on the function just before